### ISSUE TITLE
Coverage fails for GraphQLConfigs

### ISSUE THREAD

**Issue by dpeek** [graphql/graphql-js#857]
As mentioned at the end of #248, Flow coverage seems to fail on the definitions for GraphQL config objects. Type checking works, but the code is marked as uncovered in Nuclide:

![image](https://cloud.githubusercontent.com/assets/128329/26181169/0b4a8036-3b66-11e7-9c7a-871b3075fc23.png)

And to show that the definitions are being checked:

![image](https://cloud.githubusercontent.com/assets/128329/26181180/20187ce8-3b66-11e7-96e1-56caa88564ec.png)

![image](https://cloud.githubusercontent.com/assets/128329/26181191/2cfc22fc-3b66-11e7-92fc-cecb8d9ce865.png)

**Comment by dpeek** [comment]
Update: seems like it's caused by this flow issue: https://github.com/facebook/flow/issues/2748

TLDR: constructors are typed as `any` unless you explicitly set a return type of `void` (or at least that's the work around)

**Comment by leebyron** [comment]
Thanks for the report!

**Comment by dpeek** [comment]
Awesome response time :) Thanks!
